react/src/components/tabs/components/SprkTabsButton/SprkTabsButton.js

```diff
@@ -20,18 +20,21 @@ class SprkTabsButton extends Component {
       tabBtnChildren,
       tabBtnId,
       tabBtnAddClasses,
+      tabBtnAdditionalClasses,
       ariaControls,
       ariaSelected,
       isActive,
       onTabClick,
       tabBtnDataId,
       tabBtnAnalytics,
+      tabBtnAnalyticsString,
     } = this.props;
     return (
       <button
         className={classnames(
           'sprk-c-Tabs__button',
-          tabBtnAddClasses,
+          tabBtnAdditionalClasses || tabBtnAddClasses,
```

Comment:
One of the things I learned yesterday is that we can set this in the props so we don't need a check here. Check out the SprkToggle or SprkAccordion PRs to see what I mean!

Reply:
oh wow that's so much cleaner!